### PR TITLE
Developer docs for SDK X cocos 10.3.0

### PR DESCRIPTION
**Pull request by helpshift-tanuj**

### CODE REVIEW

File: docs/sdkx-cocos2dx/sdk-configuration.mdx [hunk 1]
```diff
@@ -379,6 +379,39 @@ configMap["subsequentIssuesInSameSessionConfig"] = subsequentIssuesInSameSession
...
+cocos2d::ValueMap getAPIConfig() {
+    cocos2d::ValueMap configMap;
+
+    // user specific config
+
+    // This is used for the first issue in the session
+    configMap["initialUserMessage"] = Value("INSERT YOUR INITIAL USER MESSAGE");
+
+    // This will be used for all subsequent issues in that session
+    configMap["subsequentIssuesInSameSessionConfig"] = subsequentIssuesInSameSessionConfig;
```
Comment (hs-saunved) [inline docs/sdkx-cocos2dx/sdk-configuration.mdx:395]: Please define `subsequentIssuesInSameSessionConfig` in this code block

File: docs/sdkx_android/sdk-configuration.mdx [hunk 2]
```diff
@@ -313,6 +313,44 @@ config.put("subsequentIssuesInSameSessionConfig", subsequentIssuesInSameSessionC
...
+    // config map
+    Map<String, Object> localConfig = new HashMap<>();
+
+    // ....
+    // user specific config
+    
+    // This is used for the first issue in the session
+    localConfig.put("initialUserMessage", "INSERT YOUR INITIAL USER MESSAGE");
+    // This will be used for all subsequent issues in that session
+    localConfig.put("subsequentIssuesInSameSessionConfig", subsequentIssuesInSameSessionConfig);
```
Comment (hs-saunved) [inline docs/sdkx_android/sdk-configuration.mdx:346]: Same here, please define `subsequentIssuesInSameSessionConfig` in this code block

File: docs/sdkx-unity/sdk-configuration-android.mdx [hunk 3]
```diff
@@ -231,6 +231,27 @@ public class MyGameControl : MonoBehaviour
...
+// initialise proactiveConfig collector
+
+   public class ProactiveConfigCollector : IHelpshiftProactiveAPIConfigCollector
+    {
+        public Dictionary<string, object> getLocalApiConfig()
+        {
+            Dictionary<string, object> proactiveConfig = new Dictionary<string, object>();
+            // user specific config
+            proactiveConfig.Add("initialUserMessage", "INSERT YOUR INITIAL USER MESSAGE");
+            proactiveConfig.Add("subsequentIssuesInSameSessionConfig", subsequentIssuesInSameSessionConfig)
```
Comment (hs-saunved) [inline docs/sdkx-unity/sdk-configuration-android.mdx:247]: Same here (check - I am not quite sure about this one).

**Review by sumeet-helpshift — APPROVED** [review]

File: docs/sdkx-cocos2dx/getting-started.mdx [hunk 4]
```diff
@@ -62,12 +71,38 @@ The Helpshift SDK X Cocos package contains the following files:
...
 1.  Copy the `Helpshift.aar` file from the `android/proj.android-studio/libs` present inside plugin folder to the `libs` directory of your android project.
 
 2.  Add dependency to Helpshift SDK in your `app/build.gradle` file.
 
+<Tabs>
+<TabItem value=">= 10.3.0" label="10.3.X">
+
 ```groovy
-repositories {
+    repositories {
```
Comment (jaydeep-helpshift) [inline docs/sdkx-cocos2dx/getting-started.mdx:88]:
```suggestion
repositories {
```

File: docs/sdkx-cocos2dx/getting-started.mdx [hunk 5]
```diff
@@ -62,12 +71,38 @@ The Helpshift SDK X Cocos package contains the following files:
...
+    implementation 'androidx.appcompat:appcompat:1.0.0'
+    implementation(name: 'Helpshift', ext:'aar')
+}
+```
+
+</TabItem>
+<TabItem value="<= 10.2.X>" label="Below 10.2.0">
+
+```groovy
+    repositories {
```
Comment (jaydeep-helpshift) [inline docs/sdkx-cocos2dx/getting-started.mdx:105]:
```suggestion
repositories {
```

File: docs/sdkx-cocos2dx/impact-analysis.mdx [hunk 6]
```diff
@@ -23,6 +23,52 @@ Since all the Helpshift initialization takes place on a background thread, the H
...
+|                                         |                                                                        |
+| --------------------------------------- | ---------------------------------------------------------------------- |
+| Chat screen (`showConversation()`)      | adds `5.9` MB (This depends on number of messages on the chat screen). |
+| FAQs screen (`showFAQs()`)              | adds `5.7` MB (This depends on number of FAQ sections on the screen).  |
+| FAQ section screen (`showFAQSection()`) | adds `5.7` MB (This depends on number of FAQS in the section).         |
+| Single FAQ screen (`showFAQ()`)         | adds `5.5` MB (This depends on content of the FAQ).                    |
+
+## 10.3.0 - Android
+
+**Android**
```
Comment (jaydeep-helpshift) [inline docs/sdkx-cocos2dx/impact-analysis.mdx:51]:
```suggestion
```

File: docs/sdkx-cocos2dx/release-notes.mdx [hunk 7]
```diff
@@ -10,19 +10,38 @@ import { LatestSdkVersion, DownloadButton } from "@site/src/components/forDocs";
...
+## 10.3.0
 
 <DownloadButton
   text="Download SDK"
   link="[% sdkx-cocos2dx.plugin_url %]"
   classes="mb-4"
 />
 
-_July 24, 2023_
+_xxxxx_
```
Comment (jaydeep-helpshift) [inline docs/sdkx-cocos2dx/release-notes.mdx:21]: Please add actual release date

**Review by jaydeep-helpshift — DISMISSED** [review]

File: docs/sdkx-cocos2dx/release-notes.mdx (hunk 7 above)
Comment (IamJoyson) [reply docs/sdkx-cocos2dx/release-notes.mdx:21]: Ack will change it once we QA is done

File: docs/sdkx-cocos2dx/impact-analysis.mdx [hunk 8]
```diff
@@ -23,6 +23,51 @@ Since all the Helpshift initialization takes place on a background thread, the H
...
+| FAQs screen (`showFAQs()`)              | adds `5.7` MB (This depends on number of FAQ sections on the screen).  |
+| FAQ section screen (`showFAQSection()`) | adds `5.7` MB (This depends on number of FAQS in the section).         |
+| Single FAQ screen (`showFAQ()`)         | adds `5.5` MB (This depends on content of the FAQ).                    |
+
+## 10.3.0 - Android
+
+
+<Admonition type="info" title="Note">
+
+Tested with a demo app on Google Pixel 6a running Android 13.
```
Comment (helpshift-tanuj) [inline docs/sdkx-cocos2dx/impact-analysis.mdx:54]:
```suggestion
Tested with a demo app on Google Pixel 6 running Android 14.
```

**Review by sumeet-helpshift — APPROVED** [review]

**Review by IamJoyson — APPROVED** [review]

**Pull request merged**